hyperstart.go

```diff
@@ -122,6 +122,25 @@ type hyperstartProxyCmd struct {
 	token   string
 }
 
+// HyperPod is the structure corresponding to the format expected
+// by hyperstart to start the pod on the guest.
+type HyperPod struct {
+	Hostname   string                `json:"hostname"`
+	Containers []hyperJson.Container `json:"containers,omitempty"`
+	Interfaces []HyperNetIface       `json:"interfaces,omitempty"`
+	ShareDir   string                `json:"shareDir"`
+}
+
+// HyperNetIface is the structure corresponding to the format expected
+// by hyperstart to setup the network on the guest.
+type HyperNetIface struct {
+	Device    string `json:"device,omitempty"`
+	NewDevice string `json:"newDevice,omitempty"`
+	IPAddress string `json:"ipAddress"`
```

Comment:
there's a ipAddresses field that we currently use in our runtime to specify multiple addresses (related to my earlier question)